boards/ruuvitag/Makefile.include

```diff
@@ -1,7 +1,8 @@
-# for this board, we are using Segger's RTT as default terminal interface
-USEMODULE += stdio_rtt
-TERMPROG = $(RIOTTOOLS)/jlink/jlink.sh
-TERMFLAGS = term-rtt
+# Configure Segger's RTT when stdio_rtt is loaded
+ifneq (,$(filter stdio_rtt,$(USEMODULE)))
+  TERMPROG = $(RIOTTOOLS)/jlink/jlink.sh
```

Comment:
Seems to me this block is duplicated, as its also defined identical for the `thingy52`. I guess it would make sense to move it to `make/tools/serial.inc.mk`?!

Reply:
You are right. I moved the the common code in `serial.inc.mk` with a slightly different approach.